3.0.0/README.md

```diff
@@ -0,0 +1,327 @@
+    {
+      "id": "countries",
+      "description": "Admin 0 (country) boundaries",
+      "minzoom": 0,
+      "maxzoom": 16,
+      "fields": {
+        "iso": "ISO 3166-1 Alpha-2 code",
+        "name": "English name of the country",
+        "name_ar": "Arabic name of the country"
+      }
+    },
+    {
+      "id": "buildings",
+      "description": "A layer with an empty fields object",
+      "fields": {}
+    }
+  ]
+}
+```
+
+## 3.4 `attribution`
+
+OPTIONAL. Default: null.
+
+Contains an attribution to be displayed when the map is shown to a user. Implementations MAY decide to treat this as HTML or literal text. For security reasons, make absolutely sure that this content can't be abused as a vector for XSS or beacon tracking.
+
+```JSON
+{
+  "attribution": "<a href='http://openstreetmap.org'>OSM contributors</a>"
+}
+```
+
+## 3.5 `bounds`
+
+OPTIONAL. Default: [ -180, -85.05112877980659, 180, 85.0511287798066 ] (xyz-compliant tile bounds)
+
+The maximum extent of available map tiles. Bounds MUST define an area covered by all zoom levels. The bounds are represented in WGS 84 latitude and longitude values, in the order left, bottom, right, top. Values may be integers or floating point numbers. The minimum/maximum values for longitude and latitude are -180/180 and -90/90 respectively. Bounds MUST NOT "wrap" around the ante-meridian. If bounds are not present, the default value MAY assume the set of tiles is globally distributed.
+
+```JSON
+{
+  "bounds": [ -180, -85.05112877980659, 180, 85.0511287798066 ]
+}
+```
+
+Bounds where longitude east & west, and latitude east & west are the same are considered valid. This case typically represents a single point geometry in the entire tileset. For example:
+
+```JSON
+{
+  "bounds": [-122.34, 47.65, -122.34, 47.65]
+}
+```
+
+## 3.6 `center`
+
+OPTIONAL. Default: null.
+
+The first value is the longitude, the second is latitude (both in WGS:84 values), the third value is the zoom level as an integer. Longitude and latitude MUST be within the specified bounds. The zoom level MUST be between minzoom and maxzoom. Implementations MAY use this center value to set the default location. If the value is null, implementations MAY use their own algorithm for determining a default location.
+
+```JSON
+{
+  "center": [ -76.275329586789, 39.153492567373, 8 ]
+}
+```
+
+## 3.7 `data`
+
+OPTIONAL. Default: [].
+
+An array of data files in GeoJSON format. {z}, {x} and {y}, if present, are replaced with the corresponding integers. If multiple endpoints are specified, clients may use any combination of endpoints. All endpoints MUST return the same content for the same URL. If the array doesn't contain any entries, then no data is present in the map. *This field is for overlaying GeoJSON data on tiled raster maps and is generally [no longer used](https://github.com/mapbox/tilejson-spec/pull/43) for GL-based maps.*
+
+```JSON
+{
+  "data": [
+    "http://www.example.com/admin/data.geojson"
+  ]
+}
+```
```

Comment:
Should it be made more clear that this field (as well as fillzoom, grids, legend, template) are deprecated?

Reply:
None of those are deprecated that I can see.

Reply:
Yeah, these aren't deprecated in the specification, just deprecated on Mapbox APIs. I have plans to open tickets about removing these in v4!